pulser-core/pulser/backend/remote.py

```diff
@@ -139,6 +147,10 @@ def _fetch_result(
         """Fetches the results of a completed submission."""
         pass
 
+    @abstractmethod
+    def _query_result(self, submission_id: str) -> Mapping[str, Result | None]:
```

Comment:
Could you add a short description to this?

Reply:
nit: Just a bit clear to me
```suggestion
    def _query_job_results(self, submission_id: str) -> Mapping[str, Result | None]:
```

Reply:
Done.

fetch_result uses singular while query_job_results is now plural. Is that ok?

Reply:
Yes, let's not overthink it :)